src/components/DappBrowser/search/results/SearchResult.tsx

```diff
@@ -43,6 +43,13 @@ export const SearchResult = ({ index, goToUrl }: { index: number; goToUrl: (url:
 
   const onPress = useCallback(() => url.value && goToUrl(url.value), [goToUrl, url.value]);
 
+  const fallbackIconStyle = useAnimatedStyle(() => {
+    if (dapp.value?.iconUrl) {
+      return { display: 'none' };
+    } else {
+      return { display: 'flex' };
+    }
+  });
```

Comment:
I think it's generally better to avoid conditional returns in animated styles:

```suggestion
  const fallbackIconStyle = useAnimatedStyle(() => {
    return { display: dapp.value?.iconUrl ? 'none' : 'flex' };
  });
```